src/Html.php

```diff
@@ -1608,6 +1608,7 @@ public static function address(string|Stringable $content = '', array $attribute
      */
     public static function renderTagAttributes(array $attributes): string
     {
+        $attributes = self::filterNullAttributes($attributes);
```

Comment:
Not need remove any items with null value. Need remove only attributes.

For top-level attributes it already implemented: https://github.com/yiisoft/html/blob/ac78263df12bcc699cc0a1fbb67624f2fe34234d/src/Html.php#L1663

Need implement for other attributes (see `self::DATA_ATTRIBUTES` and `self::ATTRIBUTES_WITH_CONCATENATED_VALUES`)

Reply:
Changed to only filter attributes in ```self::DATA_ATTRIBUTES``` and ```self::ATTRIBUTES_WITH_CONCATENATED_VALUES```